commandline/WSL/install-on-server.md

```diff
@@ -12,80 +12,78 @@ ms.assetid: 9281ffa2-4fa9-4078-bf6f-b51c967617e3
+* Server Datacenter Core: B69WH-PRNHK-BXVK3-P9XF7-XD84W
+* Server Standard Core: V6N4W-86M3X-J77X3-JF6XW-D9PRV
 
-4. Unzip the file into the install directory
-Additionally, I do not know if users will need to install compression on their machine or if it is in naturally.  I tested with the following commands but there is probably an easier way:
-[System.Reflection.Assembly]::LoadWithPartialName("System.IO.Compression.FileSystem") | Out-Null
-Expand-Archive $pathToZip $targetDir
+When you're done, you should see something like this:  
+![](media/WindowsServer.png)
 
-5. Run the .exe.  This would be ubuntu.exe, fedora.exe, etc…
- 
-5. Create your UNIX username and password.  This user account can be different from, and has no relationship to, your Windows username and password. [Read more](https://msdn.microsoft.com/en-us/commandline/wsl/user_support).
+Remember to start PowerShell.
 
-You're done!  Now you can use your Linux environment.
+### Check your build number
 
-## Install default Linux environment using lxrun
-lxrun installs Ubuntu user-mode by default on top of the Windows subsystem for Linux.  
+To find your Windows Server build number, run the following in PowerShell:  
+``` PowerShell
+systeminfo | findstr /B /C:"OS Name" /C:"OS Version"
+```
 
-> Since moving to the store, we have stopped keeping this user-mode image up to date.  When you're done, run apt-get update.
+You can also confirm that your build has the Windows subsystem for Linux by running the following in PowerShell:  
+``` PowerShell
+Get-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+```
 
-1. Turn on Developer Mode  
+## Enable the Windows Subsystem for Linux
+Enable the "Windows Subsystem for Linux" optional feature and reboot.
 
-    Open **Settings** -> **Update and Security** -> **For developers**
+1. Open PowerShell as Administrator and run:
+    ``` PowerShell
+    Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+    ```
+    
+    > **Troubleshooting: Enable-WindowsOptionalFeature is not recognized**  
+    > Make sure you started PowerShell.  Windows Server Core starts cmd by default.
+
+2. Restart your computer when prompted.
 
-    Select the Developer Mode radio button  
-    ![](media/updateAndSecurity.png)
+## Install default Linux environment using lxrun
 
-2. Open a command prompt.  Run `bash`  
-    ![](media/bashShellInstall.png)
+lxrun installs Ubuntu user-mode by default on top of the Windows subsystem for Linux.  
 
-    After you have accepted the License, the Ubuntu user-mode image will be downloaded and extracted. A "Bash on Ubuntu on Windows" shortcut will be added to your start menu.
+2. Run `lxrun /install`  
```

Comment:
`lxrun /install` is only for legacy reasons. I am not sure if we should advertise it as the official way of installing on Server. If this doc is only for some testing purposes, then it's fine. Otherwise, we should go with the .appx way.